bin/os_main.tf

```diff
@@ -1,29 +1,24 @@
 # Openstack credentials and instance settings, SSH Variables and current image version
-variable username {}
 
-variable password {}
 variable auth_url {}
-variable user_domain_id {}
-variable domain_id {}
+variable username {}
+variable password {}
+variable domain_name {}
+variable tenant_name {}
```

Comment:
Our Openstack provider has made some changes to the RC file, therefore less variables are now needed to instantiate VM with both `terraform` and `packer`.